src/windows/top_toolbar.c

```diff
@@ -425,7 +427,7 @@ static void window_top_toolbar_mousedown(int widgetIndex, rct_window*w, rct_widg
 
 #ifndef DISABLE_TWITCH
 		if (_menuDropdownIncludesTwitch && gTwitchEnable) {
-			dropdown_set_checked(11, true);
+			dropdown_set_checked(13, true);
```

Comment:
I'm pretty sure that when they added the Save Game As option they forgot to increment it.